lib/gooddata/models/model.rb

```diff
@@ -86,7 +86,7 @@ module Model
       'GDC.time.day_us_noleading', # M/d/yy
     ]
 
-    GD_DATA_TYPES = ['BIGINT', 'DOUBLE', 'INTEGER', 'INT', /^VARCHAR\(\d{1,4}\)$/i, /^DECIMAL\(\d{1,3},\s*\d{1,3}\)$/i]
+    GD_DATA_TYPES = ['BIGINT', 'DOUBLE', 'INTEGER', 'INT', /^VARCHAR\(\d{1,4}\)$/i, 'VARCHAR(10000)', /^DECIMAL\(\d{1,3},\s*\d{1,3}\)$/i]
```

Comment:
I don't know how to write a regex which will match values from 1 to 10000. Could you help me, @panjan ?